schedule/yast/sle/flows/default_x86_64.yaml

```diff
@@ -38,8 +38,8 @@ security: []
 default_systemd_target: []
 installation_settings:
   - installation/launch_installation
-installation:
   - installation/confirm_installation
+installation:
```

Comment:
this is fine as it was, `confirm_installation` is not related with `installation_settings`.
I suspect you did this because of some test that do not perform installation and you needed to split those actions, you can do `installation: []` in those cases like `releasenotes_origin+unregistered` we just need to stop in some point without installing, so we can modify that if that was your idea.